packages/EasyTest/src/Bridge/PhpUnit/Extension/MailerMessageLoggerListenerPhpUnitExtension.php

```diff
@@ -5,14 +5,44 @@
 namespace EonX\EasyTest\Bridge\PhpUnit\Extension;
 
 use EonX\EasyTest\Bridge\Symfony\Mailer\EventListener\MailerMessageLoggerListenerStub;
+use PHPUnit\Event\Test\Finished as TestFinishedEvent;
+use PHPUnit\Event\Test\FinishedSubscriber as TestFinishedSubscriber;
+use PHPUnit\Event\TestRunner\Started as TestRunnerStartedEvent;
 use PHPUnit\Runner\AfterTestHook;
+use PHPUnit\Runner\Extension\Extension;
+use PHPUnit\Runner\Extension\Facade;
+use PHPUnit\Runner\Extension\ParameterCollection;
+use PHPUnit\TextUI\Configuration\Configuration;
 
-final class MailerMessageLoggerListenerPhpUnitExtension implements AfterTestHook
-{
-    public function executeAfterTest(string $test, float $time): void
+if (\class_exists(TestRunnerStartedEvent::class)) {
+    /**
+     * PHPUnit >= 10.
```

Comment:
Do we need to update the readme for PHPUnit 10?

Reply:
Nice catch, thanks. I've updated, but `usage.md`, not `readme.md`

Reply:
Thanks